server/models/Comment.js

```diff
@@ -24,11 +24,18 @@ module.exports = (sequelize, DataTypes) => {
       allowNull: false,
       defaultValue: 0,
     },
+    usersWhoUpvoted: {
+      type: DataTypes.TEXT,
```

Comment:
so lets say user1 and user2 upvote the comment, it would be stored as `"user1, user2"`?

Reply:
yes, in particular we would use the primary key and have no spaces